turkish_lm_tuner/tr_datasets.py

```diff
@@ -156,7 +156,11 @@ def convert_sts_label(label):
             try:
                 return(float(label.strip()))
             except:
-                return 0
+                try:
+                    return(float(label))
+                except:
+                    return 0
+
```

Comment:
Doesn't `float(label.strip())` cover `float(label)` already?

Did we check the outputs to see if they contain similarity scores where they cannot immediately cast to float?

Reply:
When we use conditional generation, the output is a string, so `float(label.strip())` works. When we use classification, the output is a number, so the `strip()` function naturally gives an error, so I try converting into a float.

Reply:
I see. Rather than handling it with exceptions, I suggest checking its type to choose the conversion. It would be much clearer.